recipes/strainy/meta.yaml

```diff
@@ -27,6 +27,7 @@ requirements:
   - python >=3.8
   - samtools >=1.14
   - bcftools >=1.14
+  - minimap2
```

Comment:
I wonder whether this recipe is really `noarch` - it uses dependencies which provide binaries for specific platforms.

Reply:
Actually it should be OK!
If a user tries to install strainy on an architecture that does not provide some dependency then the problem is not in strainy but in the dependency.